crowdin.yaml.tmpl

```diff
@@ -0,0 +1,19 @@
+# Template for crowdin.com translation service
+project_identifier: django-formidable
+api_key: CROWDIN-API-KEY
```

Comment:
yes ; that's because crowdin is using a static configuration file. Can't live without this, here.